extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -292,12 +293,11 @@ export class AmpAnalytics extends AMP.BaseElement {
       const children = this.element.children;
       if (children.length == 1) {
         const child = children[0];
-        if (child.tagName.toUpperCase() == 'SCRIPT' &&
-            child.getAttribute('type').toUpperCase() == 'APPLICATION/JSON') {
+        if (isJsonScriptTag(child)) {
           inlineConfig = JSON.parse(children[0].textContent);
         } else {
           user.error(this.getName_(), 'The analytics config should ' +
-              'be put in a <script> tag with type=application/json');
+              'be put in a <script> tag with type="application/json"');
```

Comment:
this is the `analytics.js` file ( just refactored something common between analytics and vega )